cylc/flow/scheduler.py

```diff
@@ -1921,3 +1925,58 @@ def _get_events_conf(self, key, default=None):
         """Return a named [cylc][[events]] configuration."""
         return self.suite_event_handler.get_events_conf(
             self.config, key, default)
+
+    def get_entire_workflow(self):
+        """Return the entire workflow.
+
+        Returns:
+            PbEntireWorkflow: the entire workflow.
+        """
+        return self.ws_data_mgr.get_entire_workflow(self.get_workflow())
+
+    def get_workflow(self):
```

Comment:
If I wanted to use `WsDataMgr` but did not want to bother creating a `Scheduler`, then all that I need is to have a function somewhere that creates a `PbWorkflow`. It makes mocking and writing unit tests normally simpler.